### PR TITLE
Update to latest bigtable with fixes for import group cache

### PR DESCRIPTION
**Pull request by shifucun**
Most diff looks reasonable. There are a few diffs from know issues:

1. flaky provenance
2. triples not stable when set limit
3. stat var name string are not with sorted pvs

### CODE REVIEW

File: internal/server/placepage/golden/get_place_page_data/county.sample.json [hunk 1]
```diff
@@ -24270,7 +24342,7 @@
           "metadata": {
             "importName": "NASA_NEXDCP30_StatVarSeriesAggr",
             "provenanceUrl": "https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-dcp30",
-            "measurementMethod": "dcAggregate/NASA_Mean_CCSM4",
```
Comment (pradh) [inline internal/server/placepage/golden/get_place_page_data/county.sample.json:24273]: This change was made a while back and I also remember seeing similar diff before... so I don't understand t his one now..

File: internal/server/placepage/golden/get_place_page_data/IG_zuid-nederland.sample.json [hunk 2]
```diff
@@ -8705,18 +8708,6 @@
         },
         "DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45": {
           "val": {
-            "2006-01": 0,
```
Comment (pradh) [inline internal/server/placepage/golden/get_place_page_data/IG_zuid-nederland.sample.json:8708]: This is intended. To drop diffs wrt 2006 dates

File: internal/server/placepage/golden/get_place_page_data/IG_tha.sample.json [hunk 3]
```diff
@@ -4959,7 +4962,7 @@
             "2100-12": 2.567841
           },
           "metadata": {
-            "importName": "NASA_NEXGDDP_Country_StatVarSeriesAggr",
+            "importName": "NASA_NEXGDDP_Country_AggrDiffStats",
```
Comment (pradh) [inline internal/server/placepage/golden/get_place_page_data/IG_tha.sample.json:4965]: This change is intended with cmip5/cmip6 refresh.

File: internal/server/node/golden/get_triples/IG_limit.json [hunk 4]
```diff
@@ -67,8 +67,8 @@
       "provenanceId": "dc/5l5zxr1"
     },
     {
-      "subjectId": "geoId/4212296",
-      "subjectName": "Central City",
+      "subjectId": "geoId/3425800",
+      "subjectName": "Garwood",
```
Comment (pradh) [inline internal/server/node/golden/get_triples/IG_limit.json:71]: Was this flaky before?  Or is it flaky after the new triple format?  @Spaceenter

File: internal/server/node/golden/get_triples/IG_place_type.json [hunk 5]
```diff
@@ -5581,7 +5581,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/pnqh772"
```
Comment (pradh) [inline internal/server/node/golden/get_triples/IG_place_type.json:5584]: Might take some time, but I will try to debug this...

**Review by pradh — APPROVED** [review]
+1 that this generally looks good.

File: internal/server/placepage/golden/get_place_page_data/county.sample.json (hunk 1 above)
Comment (shifucun) [reply internal/server/placepage/golden/get_place_page_data/county.sample.json:24273]: See the file below. The base cache mode was commented out so the change was not applied before.

**Pull request merged**